meta/main.yml

```diff
@@ -14,10 +14,15 @@ galaxy_info:
         - 7
     - name: Ubuntu
       versions:
-        - all
+        - trusty
```

Comment:
The role currently supports trusty, but I think I'll remove that support since trusty is no longer receiving LTS support from Canonical. But this can remain for now.